atomix/cluster/src/main/java/io/atomix/raft/impl/RaftContext.java

```diff
@@ -162,7 +162,7 @@ public RaftContext(
     this.logReader = raftLog.openReader(1, RaftLogReader.Mode.ALL);
 
     // Open the snapshot store.
-    this.snapshotStore = storage.getSnapshotStore();
+    this.persistedSnapshotStore = storage.getPersistedSnapshotStore();
```

Comment:
I had before Snapshot but I just wanted to make it clear that this is a persisted snapshot on using it otherwise we might could also have implementations we do in memory snapshots